.devcontainer/Dockerfile.plugin

```diff
@@ -1,2 +1,2 @@
-FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:16 as build
-COPY --from=openshift/origin-cli:latest /usr/bin/oc /usr/local/bin/oc
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:20 as build-image
```

Comment:
> Why do we need to change `build` to `build-image` ?

We can keep build as it is, but `build-image` clearly indicates that this stage is specifically for creating an image for the build. For clarity, readability, and consistency with the Jolokia API Server project, I thought it would be better to update it.

Reply:
If you find it easier to read, that's a good enough reason to me.